package.json

```diff
@@ -500,6 +500,16 @@
               "onHover",
               "onSelection"
             ]
+          },
+          "kaoto.colorScheme": {
+            "type": "string",
+            "markdownDescription": "Choose the color scheme for the UI. Can be either `auto`, `light`, or `dark`",
```

Comment:
I think we should mention:
* what means auto (picking the category them of the VS Code instance)
* that the editor must be closed and reopen to apply the change

Reply:
done

Reply:
if I'm not mistaken, it is still not mentioned int he visible part to the users, in the description of the setting